mcs/class/Mono.Security/Mono.Security.X509/PKCS12.cs

```diff
@@ -383,8 +383,13 @@ private void Decode (byte[] data)
 
 				byte[] authSafeData = authSafe.Content [0].Value;
 				byte[] calculatedMac = MAC (_password, macSalt.Value, _iterations, authSafeData);
-				if (!Compare (macValue, calculatedMac))
-					throw new CryptographicException ("Invalid MAC - file may have been tampered!");
+				if (!Compare (macValue, calculatedMac)) {
```

Comment:
No, I was attempting to generate another PR for a problem with certmgr
import operations and it seems to have been combined with an earlier one. So
ignore it for this purpose but I will be attempting to get another PR to
address the second problem.

In mcs/class/Mono.Security/Mono.Security.X509/PKCS12.cs
<https://github.com/mono/mono/pull/2378#discussion_r49197809> :
> @@ -383,8 ++383,13 @@ private void Decode (byte[] data)
>  
>  				byte[] authSafeData = authSafe.Content [0].Value;
>  				byte[] calculatedMac = MAC (_password, macSalt.Value, _iterations,
authSafeData);
> -				if (!Compare (macValue, calculatedMac))
> -					throw new CryptographicException ("Invalid MAC - file may have been
tampered!");
> ++				if (!Compare (macValue, calculatedMac)) {
Hey Neal,

Are these changes supposed to be part of this pull request?